akka-docs/src/main/paradox/java/distributed-pub-sub.md

```diff
@@ -125,27 +148,43 @@ can explicitly remove entries with `DistributedPubSubMediator.Remove`.
 
 An example of a destination actor:
 
-@@snip [DistributedPubSubMediatorTest.java]($akka$/akka-cluster-tools/src/test/java/akka/cluster/pubsub/DistributedPubSubMediatorTest.java) { #send-destination }
+Scala
+:  @@snip [DistributedPubSubMediatorSpec.scala]($akka$/akka-cluster-tools/src/multi-jvm/scala/akka/cluster/pubsub/DistributedPubSubMediatorSpec.scala) { #send-destination }
 
-Subscriber actors can be started on several nodes in the cluster, and all will receive
-messages published to the "content" topic.
```

Comment:
Wrong sentence for Java. The Java page repeats the previous "Subscriber" sentences, but it has to be "Destination".